Ryujinx.HLE/HOS/Horizon.cs

```diff
@@ -283,25 +303,26 @@ public void LoadKip(string kipFile)
 
             XciPartition securePartition = xci.OpenPartition(XciPartitionType.Secure);
 
-            foreach (DirectoryEntry ticketEntry in securePartition.EnumerateEntries("*.tik"))
+            foreach (DirectoryEntryEx ticketEntry in securePartition.EnumerateEntries("/", "*.tik"))
             {
-                Ticket ticket = new Ticket(securePartition.OpenFile(ticketEntry.FullPath, OpenMode.Read).AsStream());
+                Result rc = securePartition.OpenFile(out IFile ticketFile, ticketEntry.FullPath, OpenMode.Read);
```

Comment:
```suggestion
                Result resultCode = securePartition.OpenFile(out IFile ticketFile, ticketEntry.FullPath, OpenMode.Read);
```

and add an empty line after.